config/locales/land_transaction_tax.en.yml

```diff
@@ -3,16 +3,16 @@ en:
     meta:
       title: "Land Transaction Tax (LTT) - Welsh Stamp Duty Calculator"
       description: "Use our Land Transaction Tax (LTT) calculator to calculate how much LTT (formerly Stamp Duty) you'll need to pay on your home in Scotland"
-      canonical_url: "https://www.moneyadviceservice.org.uk/en/tools/house-buying/land-transaction-tax-calculator-wales"
+      canonical_url: "https://www.moneyhelper.org.uk/en/tools/house-buying/land-transaction-tax-calculator-wales"
```

Comment:
@Venetia-maps

Reply:
this is not right should be  https://www.moneyhelper.org.uk/en/homes/buying-a-home/land-transaction-tax-calculator-wales